### PR TITLE
Change order of print statement

### PR DESCRIPTION
**Pull request by victzhan**

### CODE REVIEW

**Review by svenvh — COMMENTED** [review]
LGTM, thanks.

Does `test_imagearraycopy.cpp` have the same problem?  If so, you might as well fix that too.

**Comment by victzhan** [comment]
Hi, change as been made for test_imagearraycopy.cpp as well.

File: test_conformance/basic/test_imagearraycopy.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ int test_imagearraycopy_single_format(cl_device_id device, cl_context context, c
   clMemWrapper      image, buffer;
   int        img_width = 512;
   int        img_height = 512;
-  size_t    elem_size;
+  size_t    elem_size;
```
Comment (svenvh) [inline test_conformance/basic/test_imagearraycopy.cpp:32]: This change shouldn't be necessary and causes the formatting check to fail now.   Other than this, changes in this file look good to me.

Comment (victzhan) [reply test_conformance/basic/test_imagearraycopy.cpp]: Sorry, that should be a typo. Removed the white space.

**Review by svenvh — APPROVED** [review]

**Review by EwanC — APPROVED** [review]

**Review by lakshmih — APPROVED** [review]

**Comment by EwanC** [comment]
Merging this as it has 3 approvals

**Pull request merged**